libraries/libexosphere/include/exosphere/secmon/secmon_configuration_context.arch.arm64.hpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2018-2020 Atmosphère-NX
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include <vapours.hpp>
+#include <exosphere/pkg1.hpp>
+#include <exosphere/se.hpp>
+#include <exosphere/secmon/secmon_monitor_context.hpp>
+
+namespace ams::secmon {
+
+    struct ConfigurationContext {
+        union {
+            SecureMonitorConfiguration secmon_cfg;
+            u8 _raw_exosphere_config[0x80];
+        };
+        union {
+            EmummcConfiguration emummc_cfg;
+            u8 _raw_emummc_config[0x120];
+        };
+        u8 sealed_device_keys[pkg1::KeyGeneration_Max][se::AesBlockSize];
+        u8 sealed_master_keys[pkg1::KeyGeneration_Max][se::AesBlockSize];
+        pkg1::BootConfig boot_config;
+        u8 rsa_private_exponents[4][se::RsaSize];
+        union {
+            u8 _misc_data[0xFC0 - sizeof(_raw_exosphere_config) - sizeof(_raw_emummc_config) - sizeof(sealed_device_keys) - sizeof(sealed_master_keys) - sizeof(boot_config) - sizeof(rsa_private_exponents)];
+        };
+        /* u8 l1_page_table[0x40]; */
```

Comment:
Was this commented out by mistake?

Reply:
Nope. That data is there but not accessible via the struct.

This is why sizeof() is 0xFC0, not 0x1000 :)